<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="returnToMenuButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>170</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="poweronButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Power On</string>
    </property>
   </widget>
   <widget class="QListWidget" name="menu">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NEW SESSION</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SESSION LOG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TIME AND DATE</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="contactLight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>16</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 74, 135);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="treatLight">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>16</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="contactLostLight">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>16</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(191, 64, 64);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QListWidget" name="sessionLog">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>256</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="batteryIndicator">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>530</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(165, 29, 45);</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="rechargeButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Recharge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="poweroffButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Off</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
